src/dev/flang/tools/docs/Html.java

```diff
@@ -387,16 +412,16 @@ private String redefines0(AbstractFeature af)
   private String mainSection(Map<AbstractFeature.Kind, TreeSet<AbstractFeature>> map, AbstractFeature outer)
   {
     // Type Parameters
-    var typeParameters = new TreeSet<AbstractFeature>();
+    var typeParameters = new LinkedList<AbstractFeature>();
     typeParameters.addAll(map.getOrDefault(AbstractFeature.Kind.TypeParameter, new TreeSet<AbstractFeature>()));
     typeParameters.addAll(map.getOrDefault(AbstractFeature.Kind.OpenTypeParameter, new TreeSet<AbstractFeature>()));
     typeParameters.addAll(outer.typeArguments());
 
     // Fields
-    TreeSet<AbstractFeature> fields =  new TreeSet<AbstractFeature>();
+    var fields =  new LinkedList<AbstractFeature>();
```

Comment:
why linkedlist? why not plain list/arraylist?

Reply:
does it make a difference?

Reply:
Almost everything makes a difference in code unfortunately, sometimes even whitespace... :-D
We use a LinkedList when we need a queue, add at end, remove from beginning. Do you need this here? Otherwise I would suggest List.
Performance implications are irrelevant but consistency is quite important in a big code base. So this would be my main argument why to use a List instead of a LinkedList.